core/src/test/java/apoc/algo/PathFindingTest.java

```diff
@@ -58,6 +56,10 @@ public void setUp() throws Exception {
    		TestUtil.registerProcedure(db, PathFinding.class);
    	}
 
+    @After
```

Comment:
should this be AfterClass instead as we don't create/start the db per test method. and doing it per method would explode the test runtime.